.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+# Monolithic build. Most of the stages of the pipeline are combined into one task
+# because that is how the POM file is written.
+# Some goals, or at least areas to explore are:
+#   - Use Circle cache to move artifacts from one Job to the next in the Workflow
+#   - Build the artifacts in the first job, then run tests (note: no tests in an API repo) and
+#       OWASP, Sonar, and other analysis.
+#       processes in subsequent Jobs, in parallel.
+#   - Figure out how Circle CI likes to do Git tags, updating versions in POM files
+#   - Deploy the artifacts to a repository
+#   How committed are we to OWASP plugin? Will Snyk or Sonar or another service do it better?
```

Comment:
I'd like to keep `dependency-check-maven` in the CI build. I don't think it hurts anything.

Reply:
I'd like to see it go, eventually. It is slow to run and downloads from a database with spotty availability. I spoke with the security architect this morning and discovered that there are a lot of CVEs it is missing. My hope is that it can be replaced by a service that is more reliable, more thorough, and faster. That said, I would not want to remove the Dependency Check plugin until the security team told us to do it.

Reply:
Until it can be replaced, can we at least keep around the Jenkins CI check that can use our cache of CVEs?

Reply:
I have not gotten rid of it. The dependency check plugin is running as a part of the Circle CI build.

Reply:
CircleCI doesn't use the cache CVEs, right? Won't that be an issue?